languages/en/timers.genie

```diff
@@ -347,6 +347,12 @@ when_brushing_teeth = {
   'after i brush my teeth' => 'after i brush my teeth';
 }
 
+on = {
+  'on' => 'on';
+  'when it\'s' => 'when it\'s';
```

Comment:
space between `it` and `'s`
(also, the difference between "it is" and "it's" should probably be handled in postprocessing)

Reply:
Okay will add the space.
Do you mean contractions are already handled or should we update the code for that somewhere? 
So I should keep 'when it \'s' and remove 'when it is', or the other way around?